packages/n/nana/xmake.lua

```diff
@@ -32,8 +36,9 @@ package("nana")
         end
         import("package.tools.cmake").install(package, configs)
 
-        if is_plat("windows") then
-            os.cp("include", package:installdir())
+        os.cp("include", package:installdir())
+        if is_plat("linux") then
+            os.cp("**.a", package:installdir("lib"))
```

Comment:
why? cmake.install did not copy them? 

and so for shared

Reply:
Yes it didn't copy any files

Reply:
In [this document](https://github.com/qPCR4vir/nana-docs/wiki/Installation) they only write about static linking. So I think a shared build is not supported

Reply:
you can pass build path to cmake

and copy build/xx/.a files 

see https://github.com/xmake-io/xmake-repo/blob/7b44576aa1b54d67a04e4088e816d6a7143c4f0c/packages/b/box2d/xmake.lua#L17

Try to avoid recursive pattern matching